apps/developer-hub/src/components/IntegrationCard/index.tsx

```diff
@@ -22,35 +26,59 @@ const colorClasses = {
     text: "text-purple-600 dark:text-purple-400",
     hoverText: "group-hover:text-purple-600 dark:group-hover:text-purple-400",
   },
+  yellow: {
+    bg: "bg-yellow-100 dark:bg-yellow-900",
+    text: "text-yellow-600 dark:text-yellow-400",
+    hoverText: "group-hover:text-yellow-600 dark:group-hover:text-yellow-400",
+  },
 };
 
-export const IntegrationCard = ({
+export function IntegrationCard({
   href,
   icon,
   title,
   description,
   colorScheme = "blue",
-}: IntegrationCardProps) => {
+  external,
+  showArrow = true,
+}: IntegrationCardProps) {
   const colors = colorClasses[colorScheme];
+  const Wrapper = external ? "a" : Link;
+  const wrapperProps = external
+    ? { href, target: "_blank", rel: "noopener noreferrer" }
+    : { href };
 
   return (
-    <a
-      href={href}
-      className="block group bg-white dark:bg-gray-900 border border-gray-200 dark:border-gray-700 rounded-lg p-6 hover:border-gray-300 dark:hover:border-gray-600 transition-colors"
+    <Wrapper
+      {...(wrapperProps as
+        | { href: string }
+        | { href: string; target: string; rel: string })}
+      className="not-prose group no-underline grid h-full grid-rows-[auto_1fr] gap-3 rounded-xl border bg-[var(--color-fd-card)] border-[var(--color-fd-border)] p-5 md:p-6 shadow-sm outline-none transition-colors hover:border-[var(--color-fd-accent)] focus-visible:ring-2 focus-visible:ring-[var(--color-fd-accent)]"
```

Comment:
I realize this isn't newly added, but I think using tailwind for all these styles is a mistake.  We don't use tailwind in the component library or design system; we're using sass with `theme.scss` in the component library exporting the core design tokens and shared styles.

I realize it will be a challenge for you to convert this to sass, but I also think it's a bad precedence to get in the habit of writing tailwind as it will make this app harder to integrate with the core design system & component library.

Tailwaind is used internally by fuma docs so there will be some places we have to use tailwind to interface, but in general I don't think any custom components are a place where it makes sense to use tailwaind.